### ISSUE TITLE
Visibility swap issue

### ISSUE THREAD

**Issue by kjrawson** [CollegeOfTheHolyCross/dta_sufia#110]
Hey, Steven.
I am getting a new type of error when I try to make an item in the file listing visible (image included). I was trying to make "The female-impersonators" visible: https://www.digitaltransgenderarchive.net/files/s4655g713. 

Thanks!

![visibilityerror](https://cloud.githubusercontent.com/assets/15388345/26799491/e805bb02-4a03-11e7-860a-75afb1c10bf4.jpg)

**Comment by kjrawson** [comment]
![visibilityerror](https://cloud.githubusercontent.com/assets/15388345/26799560/1d92b3b0-4a04-11e7-9175-def2423dcaff.jpg)

**Comment by scande3** [comment]
Should be corrected now. The issue was there was a piece of Javascript that ran on a CSS class that had been used on that element (copied incorrectly from the single-use link menu item). That Javascript seems to no longer function correctly (likely due to a browser update since it still works on the older Chrome on my laptop).

I've removed the CSS element from this item to prevent that code from being called on it. It still exists on the Single-Use Link to File item but I don't believe we use that option and I don't see a need to fix that now.

TLDR: Swapping visibility there should work now. Fixed in commit 738c4cf5124f61f6e96d2616c619ed7824998e6c .

**Comment by kjrawson** [comment]
It worked perfectly––thank you!!
